### PR TITLE
Clarify Translation Process. Also fix branch names

### LINKED ISSUE #2305: Crowdin not documented in CONTRIBUTING.md
Hi,

The title says it all.

### PR DESCRIPTION
**Pull request by bkcsoft**
@appleboy @lunny, Do we have a rule on total % translated for translation sync?

Closes https://github.com/go-gitea/gitea/issues/2305

### CODE REVIEW

**Comment by appleboy** [comment]
I don't have any idea about the rule on total % translated. I think to sync the translations before release the next version.  LGTM

**Review by appleboy — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -51,6 +52,10 @@ Pull requests should only include `vendor/` updates if they are part of the same
 
 The `vendor/` update needs to be justified as part of the PR description, and must be verified by the reviewers and/or merger to always reference an existing upstream commit.
 
+## Translation
+
+We do all translation work inside [CrowdIn](https://crowdin.com/project/gitea). The only translation that is maintained in this git repository is [`en_US.ini`](https://github.com/go-gitea/gitea/blob/master/options/locale/locale_en-US.ini) and is synced regularily to CrowdIn. Once a translation has reached A SATISFACTORY PERCENTAGE it will be synced back into this repo and included in the next released version.
```
Comment (bkcsoft) [inline CONTRIBUTING.md]: `A SATISFACTORY PERCENTAGE`  should either be decided or changed 😉

Comment (ethantkoenig) [inline CONTRIBUTING.md]: Minor, but the website's name seems to be stylized as `Crowdin`, not `CrowdIn` (https://www.google.com/search?q=crowdin)

**Review by ethantkoenig — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (bkcsoft) [reply CONTRIBUTING.md]: Done 🙂

**Comment by ethantkoenig** [comment]
LGTM

**Review by ethantkoenig — APPROVED** [review]

**Pull request merged**